src/middleware/packages/webhooks/service.js

```diff
@@ -92,6 +92,9 @@ const WebhooksService = {
     webhooks: {
       name: '*',
       async process(job) {
+        // TODO: This might be a security issue since it can run arbitrary actions of this service.
+        //  @srosset81 -- I didn't find a place where this service is used.
+        //   Should we deprecate this (at least temporarily)?
```

Comment:
It's not used anymore at the moment. But the action is set by the WebhookService so I don't see how it's a security issue. See https://semapps.org/docs/middleware/webhooks#usage

Reply:
Ah, my bad. I didn't see that above :)